docs/helm-chart/using-additional-containers.rst

```diff
@@ -22,7 +22,7 @@ Sidecar Containers
 ------------------
 
 If you want to deploy your own sidecar container, you can add it through the ``extraContainers`` parameter.
-You can define different containers for the scheduler, webserver and worker pods.
+You can define different containers for the scheduler, webserver, worker, triggerer, flower and Airflow k8s job pods.
```

Comment:
```suggestion
You can define different containers for the scheduler, webserver, worker, triggerer, flower, create user Job and migrate database Job Pods.
```

I think it makes sense to enumerate the jobs? Thoughts?

Reply:
@jedcunningham , sure, updated.